elfi/methods/methods.py

```diff
@@ -515,6 +544,16 @@ def _update_objective(self):
         self.objective['n_batches'] = ceil(n_batches)
         logger.debug('Estimated objective n_batches=%d' % self.objective['n_batches'])
 
+    def plot_state(self, **options):
+        displays = []
+        if options.get('interactive'):
+            from IPython import display
```

Comment:
Maybe put the import in the beginning of `infer` where it checks if `vis`was given.